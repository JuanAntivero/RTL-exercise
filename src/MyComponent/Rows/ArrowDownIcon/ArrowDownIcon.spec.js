import ArrowDownIcon from './ArrowDownIcon';
import { shallow } from 'enzyme';

describe('ArrowDownIcon', () => {
    const wrapper = shallow(<ArrowDownIcon />);
    it('should match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    })
})