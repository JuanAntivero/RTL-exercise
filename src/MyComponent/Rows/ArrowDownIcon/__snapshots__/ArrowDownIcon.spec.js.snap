// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArrowDownIcon should match snapshot 1`] = `ShallowWrapper {}`;
